analyses/mdg/mdg_plague_exploration.md

```diff
@@ -278,7 +318,7 @@ df_urb = df_urb[df_urb.urban_area_weighted.notnull()]
 df_urb = df_urb.loc[df_urb.urban_area_weighted & (df_urb.clinical_form == "PP")]
 
 #group by date
-df_date_urb=plague_group_by_date(df_urb, sel_start_date="2017-01-01", sel_end_date="2021-10-01")
+df_date_urb=plague_group_by_date(df_urb, sel_start_date="2012-01-01", sel_end_date="2021-10-01")
```

Comment:
It seems by doing this you loose 2012 (no earlier urban cases I guess), should fix this (in refactor)

Reply:
Not sure what you mean, seems to work for me. I use it to ensure that we extend the time series to the beginning of 2012 to show we have 0 reported cases for the entire year, otherwise it would start in 2013 whenever the first urban case is observed.